### PR TITLE
Updates init and urls to handle docs page link activation.

### LINKED ISSUE #118: Update Docs
### Environment

Include https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt/pull/347 into the cookie.

### PR DESCRIPTION
**Pull request by jvanderaa**
Fixes #118

### CODE REVIEW

**Review by glennmatthews — DISMISSED** [review]

**Comment by jvanderaa** [comment]
Just to verify @glennmatthews that the pattern on the `urls.py` would work we believe?

**Comment by glennmatthews** [comment]
It *looks* like the correct pattern to me. The usual way to test it, I believe, is to run driftmanager on an app against your branch.

**Comment by gsnider2195** [comment]
We'll need to test this in `nautobot-app-dev-example` before merging here. It could have been something that I was doing wrong but I just tried using this method for `urlpatterns` and it was failing for me.

**Comment by jvanderaa** [comment]
What part about the patterns? I made it look like others, I think golden config. This works though:

https://drive.google.com/open?id=1ZJImdtbcvoqPqOr8fHcWQ2moXC-EvsQEr5WZiHSUSC8&usp=drive_copy

**Comment by gsnider2195** [comment]
Looks like I did mess something up. It's working for me now that I tried it again.

File: nautobot-app/{{ cookiecutter.project_slug }}/{{ cookiecutter.app_name }}/urls.py [hunk 1]
```diff
@@ -1,10 +1,18 @@
 """Django urlpatterns declaration for {{ cookiecutter.app_name }} app."""
 
+from django.urls import path
+from django.templatetags.static import static
+from django.views.generic import RedirectView
+
 from nautobot.apps.urls import NautobotUIViewSetRouter
```
Comment (gsnider2195) [inline nautobot-app/{{ cookiecutter.project_slug }}/{{ cookiecutter.app_name }}/urls.py:7]:
```suggestion
from django.templatetags.static import static
from django.urls import path
from django.views.generic import RedirectView
from nautobot.apps.urls import NautobotUIViewSetRouter
```

**Review by gsnider2195 — DISMISSED** [review]

**Review by gsnider2195 — APPROVED** [review]

**Pull request merged**